romancal/assign_wcs/assign_wcs_step.py

```diff
@@ -32,6 +32,14 @@ def process(self, input):
             for reftype in self.reference_file_types:
                 log.info(f"reftype, {reftype}")
                 reffile = self.get_reference_file(input_model, reftype)
+                # Check for a valid reference file
+                if reffile == "N/A":
+                    self.log.warning("No DISTORTION reference file found")
+                    self.log.warning("Assign WCS step will be skipped")
+                    result = input_model.copy()
+                    result.meta.cal_step.assign_wcs = "SKIPPED"
+                    return result
```

Comment:
I've reverted all my changes to this once I got involved with test_regtestdata. I think this can still be done but will need a more careful working of the code. A new ticket?